sql/load_modifications.sql

```diff
@@ -26,4 +27,15 @@ CREATE TABLE reviewed_modified_names(
     reviewed text
 );
 
-\COPY reviewed_modified_names FROM 'data/ipis_modified_names.csv' DELIMITER ',' CSV HEADER;
\ No newline at end of file
+\COPY reviewed_modified_names FROM 'data/ipis_modified_names.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS modifications_by_ID;
+CREATE TABLE modifications_by_ID as (
+    SELECT uid as old_uid, NULL as uid, field, old_value, new_value, editor, date, notes from modifications WHERE field = 'AGREEMENT'
+);
+
+DROP TABLE IF EXISTS modifications_by_value;
+CREATE TABLE modifications_by_value as (
+    SELECT  DISTINCT NULL as uid, field, old_value, new_value from modifications WHERE field <> 'AGREEMENT'
```

Comment:
Not sure exactly what you mean, where do the two fields of `PARCELNAME` and `SNAME` go? And how would it avoid the problem of one common recode changing many values unintentionally?

Reply:
Just to make sure, so far, the modification file will only contain 3 fields Parcelname, Sname, and agreement, right? And there won't be new fields added in the future modification file?

Reply:
> I wonder if this should instead be explicitly the two fields (name, sname?). Worry slightly about defaulting to this option which seems slightly more fraught than the more specific uid one. If I put in one modification for a new field for a common value, it could end up changing a lot unintentionally.

This logic has changed a little but I think @abrieff question still stands. 

I would put this responsibility on the editor of the modification table i.e. if they expect the value to have greater impact then they would need to specify all the uid they want the changes applied to. Otherwise, they are confirming they want the changes to be applied to all values matched for that field.

Reply:
@SashaWeinstein Sorry, I meant instead of WHERE field <> 'AGREEMENT', something like WHERE field IN ('PARCELNAME', 'SNAME'). I was thinking of the scenario in which somebody adds a correction say to USETYPE, with a uid, expecting it to correct just that row. In the previous logic, it would automatically do it by value, which might be unexpected behavior that could affect many rows. I think with the new logic, the risk of something like that was minimized, although there's still some responsibility on the editor of the table to understand what not putting a uid will mean.